src/main/java/com/aws/greengrass/mqttbridge/MQTTBridge.java

```diff
@@ -184,4 +188,16 @@ public void shutdown() {
             ioTCoreClient.stop();
         }
     }
+
+    private boolean skipUpdatingMqttTopicMapping(WhatHappened whatHappened, Node node) {
+        if (whatHappened == WhatHappened.timestampUpdated || whatHappened == WhatHappened.interiorAdded) {
```

Comment:
nit: I think this could be moved up a level, because these exclusions aren't necessarily specific to mqtt mapping

Reply:
I feel having the skip conditions together would be better.